README.md

```diff
@@ -61,6 +61,11 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 ### Javascript
 ([Go to top](#awesome4girls---))
 
+  - [Js4Girls](http://webschool.io/js4girls/) - Loc: Brazil - - *Loc:* <img src="https://upload.wikimedia.org/wikipedia/en/0/05/Flag_of_Brazil.svg" alt="Brazil" width="30"> - *Lang: PT-BR only*
+  
+  A Brazilian initiative to provide free javascript classes to women in a male-free environment.
+
+
```

Comment:
There is an extra empty line here. I think it is causing the build fail

Reply:
Yes we have to, create a few more issues regarding that.